src/new/cli.ts

```diff
@@ -30,13 +30,29 @@ const createProject = (): CommandModule<CommandModuleArgs, any> => {
 				type: "string",
 				alias: "d",
 			})
+			.option("experimental", {
+				describe: "Use experimental traspile with swc",
+				type: "boolean",
+				default: false,
+			})
```

Comment:
I think this could be a little more generic, something in the lines of:

_"Use experimental, not battle-tested, features of ExpressoTS"_

Reply:
We might even list what are the current set of experimental features or something like this